libs/designsystem/card/src/card.component.spec.ts

```diff
@@ -45,36 +49,75 @@ describe('CardComponent', () => {
     });
   });
 
-  describe('with mode attribute', () => {
+  describe('with flat variant attribute', () => {
+    let cardElement: HTMLElement;
+
     beforeEach(() => {
-      spectator = createHost('<kirby-card mode="flat"></kirby-card>');
+      spectator = createHost('<kirby-card variant="flat"></kirby-card>');
+      cardElement = spectator.queryHost('kirby-card');
     });
 
-    it('should create', () => {
-      expect(spectator.component).toBeTruthy();
+    it('should have no box-shadow', () => {
+      expect(cardElement).toHaveComputedStyle({
+        'box-shadow': 'none',
+      });
+    });
+  });
+
+  describe('with outlined variant attribute', () => {
+    let cardElement: HTMLElement;
+
+    beforeEach(() => {
+      spectator = createHost('<kirby-card variant="outlined"></kirby-card>');
+      cardElement = spectator.queryHost('kirby-card');
+    });
+
+    it('should have default styles', () => {
+      expect(cardElement).toHaveComputedStyle({
+        'box-shadow': 'none',
+        'background-color': 'transparent',
+        'outline-color': OUTLINE_COLOR,
+        'outline-style': 'solid',
+        'outline-width': '1px',
+      });
     });
   });
 
-  describe('with mode attribute and nested Ionic component', () => {
+  describe('with outlined variant attribute and flagged header', () => {
+    let cardElement: HTMLElement;
+
     beforeEach(() => {
       spectator = createHost(
-        `<kirby-card mode="flat">
-           <ion-badge>Test</ion-badge>
-         </kirby-card>`
+        '<kirby-card variant="outlined"><kirby-card-header flagged="warning"></kirby-card-header></kirby-card>'
       );
+      cardElement = spectator.queryHost('kirby-card');
     });
 
-    it('should create', () => {
-      expect(spectator.component).toBeTruthy();
-    });
+    it('should have default styles', () => {
+      const contentWrapperElement = spectator.queryHost('.content-wrapper');
+
+      expect(contentWrapperElement).toHaveComputedStyle({
+        'border-block-end-color': OUTLINE_COLOR,
+        'border-block-end-style': 'solid',
+        'border-block-end-width': '1px',
+
+        'border-inline-start-color': OUTLINE_COLOR,
+        'border-inline-start-style': 'solid',
+        'border-inline-start-width': '1px',
+
+        'border-inline-end-color': OUTLINE_COLOR,
+        'border-inline-end-style': 'solid',
+        'border-inline-end-width': '1px',
 
-    it('should create nested Ionic component with default Ionic mode', async () => {
-      const document = spectator.query<HTMLElement>('html', { root: true });
-      const ionicGlobalMode = document.getAttribute('mode');
-      const ionBadgeElement = spectator.query<HTMLElement>('ion-badge');
-      expect(ionBadgeElement).toBeTruthy();
-      await TestHelper.whenReady(ionBadgeElement);
-      expect(ionBadgeElement).toHaveClass('ios', ionicGlobalMode);
+        'border-bottom-left-radius': '16px',
+        'border-bottom-right-radius': '16px',
```

Comment:
Similar to the color you can use `DesignTokenHelper.size()` here instead of pixel values.